helpers/loadENV.js

```diff
@@ -1,8 +1,9 @@
 window.API_BASE_URL = 'https://api.realdevsquad.com';
 
 if (
-  window.location.hostname !== 'dashboard.realdevsquad.com' &&
-  window.location.hostname !== 'localhost'
+  (window.location.hostname !== 'dashboard.realdevsquad.com' &&
+    window.location.hostname === 'localhost') ||
+  window.location.hostname === 'dev.realdevsquad.com'
```

Comment:
If the intended behavior is to align with the previous logic, then this condition must be

 `
(window.location.hostname !== 'dashboard.realdevsquad.com' &&
    window.location.hostname !== 'localhost') ||
  window.location.hostname === 'dev.realdevsquad.com'
`

if the current logic is intended then it can be simplified to

`
window.location.hostname === 'localhost' ||
  window.location.hostname === 'dev.realdevsquad.com'
`

Reply:
can i just change it 
`window.API_BASE_URL = 'https://api.realdevsquad.com';
if (window.location.hostname!== 'dashboard.realdevsquad.com') {
  window.API_BASE_URL = 'https://staging-api.realdevsquad.com';
}`
- so the prod api will use only for the main prod website